### PR TITLE
Wire connector UI modifications

### PR DESCRIPTION
**Pull request by M-JULIANI**
### Description

This PR updates the resources and styles related to connectors/connector pins. Minor UI changes are made to match the mock-ups & the expected UX.

*Styling of connector context menu is forthcoming.

WIre-UI Modifications 27 09 21

image

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers
@QilongTang 
@saintentropy 

### FYIs
@SHKnudsen 
@Amoursol

### CODE REVIEW

**Comment by Amoursol** [comment]
@QilongTang @saintentropy - would be good to test performance after this PR is merged as it's expected to help 🚀

File: src/DynamoCoreWpf/ViewModels/Core/ConnectorViewModel.cs [hunk 1]
```diff
@@ -289,6 +289,7 @@ public bool ConnectorAnchorViewModelExists
             {
                 connectorAnchorViewModelExists = value;
                 RaisePropertyChanged(nameof(ConnectorAnchorViewModelExists));
+                RaisePropertyChanged(nameof(PreviewState));
```
Comment (QilongTang) [inline src/DynamoCoreWpf/ViewModels/Core/ConnectorViewModel.cs:292]: why do we raise a different property change here?

**Review by QilongTang — COMMENTED** [review]
one comment then LGTM

File: src/DynamoCoreWpf/ViewModels/Core/ConnectorViewModel.cs (hunk 1 above)
Comment (M-JULIANI) [reply src/DynamoCoreWpf/ViewModels/Core/ConnectorViewModel.cs:292]: image
@QilongTang  Actually the `RaisePropertyChanged` call is what indicates that the wire should transition to `hover` mode. Open to suggestions on how to do it differently.

Comment (QilongTang) [reply src/DynamoCoreWpf/ViewModels/Core/ConnectorViewModel.cs:292]: Ah I missed this then, make sense

**Review by QilongTang — APPROVED** [review]

**Comment by QilongTang** [comment]
The regressions reported are all package manager related so should be unrelated. Merging.

**Pull request merged**